desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -25,13 +25,24 @@
 from desktop.lib.django_util import login_notrequired, JsonResponse
 from desktop.lib.exceptions_renderable import PopupException
 from desktop.models import Document2, _get_gist_document
+from desktop.auth.backend import rewrite_user
+
+from notebook.api import _fetch_result_data, _check_status, _execute_notebook
+from notebook.models import MockRequest
+
+from useradmin.models import User
 
 from django.http import HttpResponse
 from django.utils.translation import ugettext as _
 from django.views.decorators.csrf import csrf_exempt
 
 LOG = logging.getLogger(__name__)
 
+try:
+  from prettytable import PrettyTable
+except ImportError:
+  pass
```

Comment:
Just for the record: it is often better to have a `LOG.warn('slack server: prettytable module is not installed')` to make it more obvious where it fails. We could also just print simple text as fallback later.

Reply:
Was thinking of logging it but saw most of the usage examples of using pass, so was resistant
Thanks for pointing this out @romainr! Will log it!